cl/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 # along with predator.  If not, see <http://www.gnu.org/licenses/>.
 
 # project metadata
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
I think this should be conditioned by `if(APPLE)` in order not to break the build on Ubuntu 14 LTS:
https://travis-ci.org/kdudka/predator/jobs/524427093

Reply:
Really? Support ends 2019-04-30.

Reply:
According to https://wiki.ubuntu.com/Releases support for Ubuntu 14 LTS goes EOL in April 2022.  Nevertheless I am not against migrating the Travis CI tests to a more recent OS.  According to https://sv-comp.sosy-lab.org/2019/rules.php SV-COMP runs on Ubuntu 18.04 now.